### PR TITLE
chore(build): update to php 7.3

### PR DESCRIPTION
**Pull request by adr-mo**
## Description

Upgrade to php 7.3

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — DISMISSED** [review]

**Review by sc979 — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by sc979 — DISMISSED** [review]

**Review by jeremyjaouen — APPROVED** [review]

File: unattended.sh [hunk 1]
```diff
@@ -98,6 +98,14 @@ if [ "x$?" '!=' x0 ] ; then
 fi
 print_step_end
 
+# Get the Centos version
+eval $(cat /etc/os-release | grep VERSION_ID)
```
Comment (sc979) [inline unattended.sh:102]: uname -mrs

Comment (sc979) [reply unattended.sh:102]: uname -r

Comment (adr-mo) [reply unattended.sh:102]: uname gives you information about the linux kernel version.

```
➜  ~ docker exec -ti 2d0 bash
[root@2d0b9de98725 /]# uname -r
4.19.121-linuxkit
[root@2d0b9de98725 /]# uname -mrs
Linux 4.19.121-linuxkit x86_64
```

We do want to know the version of Centos installed here

**Review by sc979 — APPROVED** [review]

File: unattended.sh (hunk 1 above)
Comment (adr-mo) [reply unattended.sh:102]: Checking the file is easier as it is bash formatted variables so a simple eval gives you the value

```
[root@2d0b9de98725 /]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

**Pull request merged**